src/types.jl

```diff
@@ -89,12 +89,21 @@ struct NERTaggedWord <: TaggedWord
     end
 end
 
+struct WikiGoldWord <: TaggedWord
```

Comment:
Maybe `NerOnlyTaggedWord` ?
We need to rethink these types at some point

Reply:
That's better. For the time being, let's change it to `NerOnlyTaggedWord`.